cluster/gce/config-default.sh

```diff
@@ -354,6 +354,9 @@ elif [[ -n "${MAX_PODS_PER_NODE:-}" ]]; then
   # Should not have MAX_PODS_PER_NODE set for route-based clusters.
   echo -e "${color_red:-}Cannot set MAX_PODS_PER_NODE for route-based projects for ${PROJECT}." >&2
   exit 1
+elif [[ "$(get-num-nodes)" -gt 100 ]]; then
```

Comment:
nit: can you change that int:
```
if [ "${ENABLE_IP_ALIASES}" = true ]; then
 ...
else
  if [[ -n "${MAX_PODS_PER_NODE:-}" ]]; then
    ..
  fi
  if [[ <your condition>]]; then
    ..
  fi
fi
```

It's easier to follow that those conditions are basically not really connected.

Reply:
Makes sense, done.